src/main/java/com/thangle/domain/book/ImportBookHelper.java

```diff
@@ -0,0 +1,61 @@
+package com.thangle.domain.book;
+
+import com.thangle.error.ForbiddenException;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.Instant;
+import java.util.*;
+
+public class ImportBookHelper {
+
+    // The Microsoft Office MIME type of Excel files
+    final public static String TYPE = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet";
+
+    public static boolean hasExcelFormat(MultipartFile file) {
+        return TYPE.equals(file.getContentType());
+    }
+
+    public static List<Book> extractDataFromInputStream(final InputStream inputStream, final UUID userId) {
+        List<Book> books = new ArrayList<Book>();
+
+        try {
+            XSSFWorkbook workBook = new XSSFWorkbook(inputStream);
```

Comment:
Dùng phép try resource


```suggestion
        try(final XSSFWorkbook workBook = new XSSFWorkbook(inputStream)) {
           // code xử lý
        }
```